### PR TITLE
Add from file [file path]

### PR DESCRIPTION
**Pull request by EngALAlfy**
I tries to use this package in foreground service but I cant do this because I can not use Path Provider in the service so I have to pass the file [sound file] path to the package

### CODE REVIEW

**Review by SPodjasek — CHANGES_REQUESTED** [review]
Why do you remove homepage & repository here? I'm not able to find any remark in docs that repository is not allowed for publishable packages, I believe it's even encouraged to place them as a link to source repository.

File: lib/flutter_ringtone_player.dart [hunk 1]
```diff
@@ -36,13 +36,16 @@ class FlutterRingtonePlayer {
...
+      String? fromFile,
       double? volume,
       bool? looping,
       bool? asAlarm}) async {
-    if (fromAsset == null && android == null && ios == null) {
+    if (fromFile == null && fromAsset == null && android == null && ios == null) {
       throw "Please specify the sound source.";
     }
-    if (fromAsset == null) {
+    if (fromFile != null){
```
Comment (SPodjasek) [inline lib/flutter_ringtone_player.dart:46]: Please apply `dart format` to fix this code formatting

**Review by SPodjasek — CHANGES_REQUESTED** [review]

**Review by SPodjasek — APPROVED** [review]

**Comment by SPodjasek** [comment]
@EngALAlfy And could you please rebase your branch on current `main`?

**Comment by EngALAlfy** [comment]
Check now

**Comment by SPodjasek** [comment]
@EngALAlfy Thank you for your contribution

**Pull request merged**